themes/bootstrap3/js/account_ajax.js

```diff
@@ -15,10 +15,10 @@ VuFind.register('account', function Account() {
     "DANGER": 3
   };
   var _accountIcons = {};
-  _accountIcons[ICON_LEVELS.NONE] = "fa fa-user-circle";
-  _accountIcons[ICON_LEVELS.GOOD] = "fa fa-bell text-success";
-  _accountIcons[ICON_LEVELS.WARNING] = "fa fa-bell text-warning";
-  _accountIcons[ICON_LEVELS.DANGER] = "fa fa-exclamation-triangle text-danger";
+  _accountIcons[ICON_LEVELS.NONE] = ["my-account", "ajax-status-none"];
```

Comment:
These should be account-status-* ("ajax" sounds like an AJAX request status icon).